x-element.js

```diff
@@ -2,8 +2,9 @@ import { asyncAppend } from '../../lit-html/directives/async-append.js';
 import { asyncReplace } from '../../lit-html/directives/async-replace.js';
 import { cache } from '../../lit-html/directives/cache.js';
 import { classMap } from '../../lit-html/directives/class-map.js';
-import { directive, html, render, svg } from '../../lit-html/lit-html.js';
```

Comment:
This was the only thing I had to change. The `directive` export no longer exists inside the `lit-html.js` file.